include/votca/tools/elements.h

```diff
@@ -42,48 +42,53 @@ namespace votca {
         class Elements {
         public:
 
+            Elements(): _filled_VdWChelpG(false),_filled_VdWMK(false),_filled_NucCrg(false),
+                        _filled_CovRad(false),_filled_Mass(false),_filled_EleNum(false),
+                        _filled_EleName(false),_filled_ElPolarizability(false),_filled_EleShort(false),
+                        _filled_EleFull(false){};
+
             /// ChelpG is a method for calculating the electrostatic potential outside
             /// of a molecule. The VdWChelpG radius in this case is specific to each
             /// element and defines the radius outside of which the electrostatic
             /// potential is calculated. The electrostatic potential within the radius
             /// is not calculated. For more information see the reference paper CHELPG
             /// paper [Journal of Computational Chemistry 11, 361, 1990]
-            const double &getVdWChelpG(std::string name);
+            double getVdWChelpG(std::string name);
 
             /// Merz-Singh-Kollman MK method is a similar method for calculating the
             /// electrostatic potential outside of a molecule. Details of the method can
             /// be found here [Singh, U. Chandra (04/01/1984). "An approach to computing
             /// electrostatic charges for molecules". Journal of computational chemistry
             /// 5 (2), 129, 1984]. The VdWMK method will return the radii used to define
             /// where the electrostatic grid starts.
-            const double &getVdWMK(std::string name);
+            double getVdWMK(std::string name);
 
             /// Return the Nuclear charges of each atom. H - 1, He - 2, Na - 3 etc...
-            const double &getNucCrg(std::string name);
+            double getNucCrg(std::string name);
```

Comment:
Do we want to go ahead and expand these function names, getNuclearCharge, getElementNumber, or do you want to just wait?

Reply:
We can do that but not in this PR.. for it we have to make a new branch in every repo